### ISSUE TITLE
Clarify publication plans

### ISSUE THREAD

**Issue by florianm** [dbca-wa/ruODK#14]
## Feature
Clarify publication

## Task
Source https://github.com/ropensci/software-review/issues/335 @maelle
```
Reg "Note: I would like to submit a paper about the package in a few weeks, 
but haven't got the manuscript ready and approved for publication just yet." 
This is also a good reason to put the issue on hold since the reviewers 
would read the short JOSS paper. What do you mean by approved?
```

**Comment by florianm** [comment]
A rudimentary JOSS-style paper is actually [included](https://github.com/dbca-wa/ruODK/blob/master/inst/joss/paper.md).
However I'd like to check scope/fit for an MEE paper first, which will both exclude prior publication to JOSS (is that right? can we have both?) and take a while longer than I'd like to hold up the ROpenSci submission. @maelle rightly mentioned that a few diagrams would help to draw the bigger picture - already working on it ;-)

The reason for not wanting to hold up ROpenSci is an ODK gathering later this year where I'd like to prepare/solidify my end (ruODK) as far as possible. I feel that the feedback from ROpenSci reviewers will help me find my blind spots and improve ruODK efficiently.

Being a public servant, writing a paper (MEE or JOSS) involves a joyful and bureaucratic approval process to make sure I'm picking the right words and not contradicting my Department's policies/positions/strategic directions. This means further delays of getting a paper written through our internal review.

So in a nutshell, ROpenSci review will help to deliver a robust, production-worthy package while the publication at MEE or JOSS will undoubtedly come but at a slightly slower pace than the ODK timeline would like.

**Comment by maelle** [comment]
Thanks for the explanation! @sckott please correct me if I'm wrong.

Indeed I think you'd need to choose between JOSS  and MEE.

For MEE, the paper doesn't have to be ready but it's better if it's more or less ready. We at rOpenSci have no influence on their deeming a paper in scope/fitting.

For JOSS sorry I realize I need to ask too, will write again here (and then we can summarize in the main thread 😉).

Awesome about the diagrams! As an outsider I am looking forward to seeing them!

**Comment by sckott** [comment]
I don't know about JOSS vs. MEE. I'd suggest emailing MEE and asking them. Maybe they'd consider JOSS similar enough to a pre-print, but I don't know.

**Comment by maelle** [comment]
Reg JOSS it'd be fine to add the short paper.md at the very end after the reviews, in which case I as editor would read it before you submit it for the expedited process at JOSS.

**Comment by florianm** [comment]
Picking up the publication plans, currently I have about 2 months of urgent end of field season work before I can realistically dedicate time to attempt an MEE paper.
